src/scss/components/_media-element.scss

```diff
@@ -12,7 +12,7 @@
 $mediaElement-img-width         : 50px;
 $mediaElement-img-width--mid    : 78px;
 $mediaElement-img-width--wide   : 67px;
-$mediaElement-img-borderRadius  : 8px;
+$mediaElement-img-borderRadius  : 2px;
```

Comment:
Did this change come from the UX team?

Reply:
https://app.zeplin.io/project/5a7af4078b4a016801348b6c/screen/5b72f088e47db8634b32b23c @DamianMullins

Reply:
Might be worth confirming with them that this with ripple out wherever there's a logo (like on home and search)